roles/iam-kms/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Set a date stamp for our key and key_alias
```

Comment:
Why not move this to `create-update` where the variable is used?

Reply:
Will do